.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: c
 
-sudo: false
+sudo: required
```

Comment:
Do you need `sudo`?
I couldn't find a command requiring `sudo` preceding it or is it because of docker?

Reply:
Some reference documentation demanded `sudo`, but I'll try again without it.